Do some rearranging and polishing of the file path set up for getting started for each example (#298)
AlexsLemonade/refinebio-examples: template/template_example.Rmd

These chunks will declare your file paths and double check that your files are in the right place. 

First we will declare our file paths to our data and metadata files, which should be in our data directory.
This is handy to do because if we want to [switch the dataset](#using-a-different-refinebio-dataset-with-this-analysis) we are using for this analysis, we will only have to change the file path here to get started. 

```{r}
# Define the file path to the data directory
data_dir <- file.path("data", {{experiment_accession}})

# Declare the file path to the gene expression matrix file, stored as `data_dir`
data_file <- file.path(data_dir, {{DATA_ACCESSION FILENAME}}) # Replace with file path to your dataset

# Declare the file path to the metadata file, stored as `data_dir`
metadata_file <- file.path(data_dir, {{METADATA_ACCESSION FILENAME}}) # Replace with file path to your metadata
```

Now that our file paths are declared, we can use the `file.exists()` function to check that the files are where we specified above. 

```{r}
# Check if the gene expression matrix file is where the file path stored in `data_file` says
file.exists(data_file)

> # Check if the metadata file is where the file path stored in `metadata_file` says

Reviewer: ```suggestion
# Check if the metadata file is at the file path stored in `metadata_file`
```

If you use the above suggestion this line can also change.
